common/model-views.cpp

```diff
@@ -798,6 +798,18 @@ namespace rs2
             }
         }
 
+        // Hack to restore "Enable Histogram Equalization" flag if needed.
+        // The flag is set to true by colorizer constructor, but setting min/max_distance options above or during
+        // restore_processing_block earlier, causes the registered observer to unset it, which is not the desired
+        // behaviour. Observer should affect only if a user is setting the values after construction phase is over.
+        auto colorizer_options = depth_colorizer->get_supported_options();
+        if( std::find( colorizer_options.begin(), colorizer_options.end(), RS2_OPTION_VISUAL_PRESET ) != colorizer_options.end() )
```

Comment:
try:
`if ( depth_colorizer->supports( RS2_OPTION_VISUAL_PRESET ) )`
instead of lines 805  + 806

Reply:
Changed. Don't know why intellisense did not show the function previously, I wasn't sure it is in the API.